components/contextmenu/integration.md

```diff
@@ -383,6 +384,200 @@ In this example, the context menu is used to select/deselect items, put an item
 }
 ````
 
+## Context Menu for a TreeView Node
+
+To integrate the context menu with the Telerik TreeView, you need to:
+
+1. Use the [`OnItemContextMenu`]({%slug treeview-events%}#onitemcontextmenu) event of the TreeView to get the current row model and show the menu
+2. Use the context menu's [`OnClick`]({%slug contextmenu-events%}#onclick) event to handle the desired operation
+
+In this example, the context menu is used to select/deselect items and delete items
+
+>caption Use a Context Menu for TreeView nodes
+
+````CSHTML
+@* Use the OnItemContextMenu event of the TreeView to show the ContextMenu for its items *@
+
+<TelerikContextMenu Data="@ContextMenuData"
+                    @ref="ContextMenu"
+                    OnClick="@((ContextMenuItem item) => ClickHandler(item))">
```

Comment:
May be the handler names should hint which component they are for. Otherwise you need to see the component declarations first. E.g. ContextMenuClickHandler or ContextClickHandler or OnContextMenuClick

Reply:
Indeed, checking with Stamo if the behavior will be the same as in the Grid. Will update the doc tomorrow morning.

Reply:
Updated handler names to ContextMenuClickHandler for all three examples.